README.md

```diff
@@ -1 +1,1146 @@
-# eip-message-transformation
\ No newline at end of file
+[![Build Status](https://travis-ci.org/sanethmaduranga/eip-message-transformation.svg?branch=master)](https://travis-ci.org/sanethmaduranga/eip-message-transformation.svg?branch=master)
```

Comment:
Should we change this to match what is in the master branch?

Reply:
Done

Reply:
This should actually point to ballerina-guides, not the personal account